File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/Review/ReviewGetDto.cs
namespace eRezervisi.Common.Dtos.Review
{
    public class ReviewGetDto
    {
        public long Id { get; set; }
        public double Rating { get; set; }
        public string? Note { get; set; }
        public long ReviewerId { get; set; }
        public string Reviewer { get; set; } = null!;
        public string? ReviewerImage { get; set; }
        public double MinutesAgo { get; set; }
        public double HoursAgo { get; set; }
        public double DaysAgo { get; set; }
    }
}

File: erezervisi_desktop/erezervisi_desktop/lib/models/responses/review/review_get_dto.dart
class ReviewGetDto {
  late num id;
  late num rating;
  String? note;
  late num reviewerId;
  late String reviewer;
  String? reviewerImage;
  late num minutesAgo;
  late num hoursAgo;
  late num daysAgo;

  ReviewGetDto({
    required this.id,
    required this.rating,
    this.note,
    required this.reviewerId,
    required this.reviewer,
    this.reviewerImage,
    required this.minutesAgo,
    required this.hoursAgo,
    required this.daysAgo,
  });

  factory ReviewGetDto.fromJson(Map<String, dynamic> json) {
    return ReviewGetDto(
      id: json['id'] as num,
      rating: json['rating'] as num,
      note: json['note'],
      reviewerId: json['reviewerId'] as num,
      reviewer: json['reviewer'],
      reviewerImage: json['reviewerImage'],
      minutesAgo: json['minutesAgo'],
      hoursAgo: json['hoursAgo'],
      daysAgo: json['daysAgo'],
    );
  }
}
